import/README.md

```diff
@@ -1,26 +1,48 @@
 # Import
 
-In this directory you'll find the best strategies to import CSV and Cypher queries into Memgraph.
+In this directory, you'll find the best strategies for importing CSV and Cypher queries into Memgraph.
 
+The cypher and csv folders contain examples of how to load a CYPHERL file and LOAD CSV via pymgclient and neo4jpython client. 
 
-## Node import speed reference
+Before running any import script, make sure the `../dataset/graph500/download_dataset.sh` is executed first, 
+It will pull all necessary data into the dataset directory needed to run the tests. 
+
+If you are running a test, for example, the node import process in pymgclient, you need to pass the dataset size: 
+
+```bash
+python3 ./cypher/pymgclient/concurrent_node_import.py small
+```
+
+For concurrent LOAD CSV, Memgraph needs to be in `IN_MEMORY_ANALYTICAL` mode. 
+
+## Test reference 
+
+Below are the numbers representing import speed in different scenarios. The tests were run on the following hardware: 
+
+CPU: AMD Ryzen 5 2600 Six-Core Processor
+RAM: 2x8GB 2133 MT/s DDR4
+
+All tests used ten concurrent database connections. All tests were made on docker containers; native versions of Memgraph will yield even better results.
+
+### Node import speed reference
 
 Nodes per second = n/s
 Peak memory pressure in megabytes = MB 
 
-TEST 1: 
+TEST 1: neo4jpython
```

Comment:
isn't the driver called just neo4j?

Reply:
Yep, which I find a bit confusing but, will rename to just neo4j.